release/README.md

```diff
@@ -1,41 +1,38 @@
-# Bank of Anthos Release Workflow
-
-This doc describes how repo maintainers can tag and push a new release.
+# Releasing Bank of Anthos
 
+This document describes how maintainers can tag and push a new release of Bank of Anthos.
 
 ## Prerequisites for tagging a release
 
-1. **Complete a team-wide Fishfood day** off the latest main commit. Divide and conquer tasks, to test and verify the user journeys below. If you encounter any bugs or docs in need of fixing, make those changes before proceeding with the release.
```

Comment:
I wonder how many times the team actually did team-wide "fish-fooding" of Bank of Anthos releases. 😅